### PR TITLE
cleaning up the config vars

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     "build": "npm run clean && babel -d ./dist ./src -s",
     "postinstall": "npm run build",
     "start": "node ./dist/index.js",
-    "heroku-prebuild": "echo $IOS_APN_KEY_BASE64 | base64 --decode > $IOS_APN_KEY",
```
Comment (Benjamindodgson) [inline package.json:9]: I don't think this is needed.

File: src/api.js [hunk 2]
```diff
@@ -44,14 +41,8 @@ const api = new ParseServer({
         teamId: IOS_TEAM_ID // The Team ID for your developer account
       },
       topic: IOS_BUNDLE_ID, // The bundle identifier associated with your app
-      production: false
```
Comment (Benjamindodgson) [inline src/api.js:47]: Using the config var

File: src/api.js [hunk 3]
```diff
@@ -62,7 +53,8 @@ const api = new ParseServer({
       'Connection',
       'Achievement',
       'Transaction',
-      'Reservation'
+      'Reservation',
+      'Moment'
```
Comment (Benjamindodgson) [inline src/api.js:57]: Adding moment to live query

File: src/api.js [hunk 4]
```diff
@@ -39,19 +36,13 @@ const api = new ParseServer({
   push: {
     ios: {
       token: {
-        key: IOS_APN_KEY,
+        key: `/${IOS_APN_KEY}`,
```
Comment (Benjamindodgson) [inline src/api.js:39]: Not sure if this is correct but this needs to be the file path. 

`push: {
  ios: {
    token: {
      key: '/file/path/to/AuthKey_XXXXXXXXXX.p8',
      keyId: "XXXXXXXXXX",
      teamId: "YYYYYYYYYY" // The Team ID for your developer account
    },
    topic: 'com.domain.appname', // The bundle identifier associated with your app
    production: false
  }
}`

File: package.json (hunk 1 above)
Comment (gcolotti) [reply package.json:9]: Yes we need this. This creates the p8 file.

File: src/api.js (hunk 4 above)
Comment (gcolotti) [reply src/api.js:39]: IOS_APN_KEY should be the file path in heroku envs. I will double check

File: src/server.js [hunk 5]
```diff
@@ -8,7 +8,6 @@ import api from './api';
 
 // Stream Webhooks
 import chatAfterEvent from './chatEventWebhooks/chatAfterEvent';
-import typeFormEvent from './chatEventWebhooks/typeFormEvent';
```
Comment (gcolotti) [inline src/server.js:11]: We don´t need this anymore?

Comment (Benjamindodgson) [reply src/server.js:11]: Nope

File: package.json (hunk 1 above)
Comment (Benjamindodgson) [reply package.json:9]: @gcolotti can you explain to me why this is needed? I didn't see any examples of this online.

Comment (gcolotti) [reply package.json:9]: Yes. Before heroku starts to build the application, it runs this prebuild command. It basically is a bash command that generates a file in the root of the project, with the IOS_APN_KEY name/path and IOS_APN_KEY_BASE64 content. If you check those envs in our server config, you will see they correspond to the p8 file we use

File: src/api.js (hunk 4 above)
Comment (gcolotti) [reply src/api.js:39]: I think we should leave it as it was

**Pull request merged**